lua/mapvote/shared/modules/config_schema.lua

```diff
@@ -39,6 +40,7 @@ local default = {
     IncludedMaps = {},
     ExcludedMaps = {},
     RTVPercentPlayersRequired = 0.66,
+    RTVPercentMulWhenOverpopulated = 0.75,
```

Comment:
Once changed to not use a mult, default this to 0 and treat 0 as the feature being disabled

Reply:
Done